test/core/tableTests.ts

```diff
@@ -68,14 +68,41 @@ describe("Tables", () => {
     assert.isNull(rows[1][0], "component at (1, 0) is null");
   });
 
-  it("can't add a component where one already exists", () => {
+  it("Add a component where one already exists creates a new group", () => {
     var c1 = new Plottable.Component.AbstractComponent();
     var c2 = new Plottable.Component.AbstractComponent();
     var c3 = new Plottable.Component.AbstractComponent();
     var t = new Plottable.Component.Table();
+
     t.addComponent(0, 2, c1);
     t.addComponent(0, 0, c2);
-    assert.throws(() => t.addComponent(0, 2, c3), Error, "component already exists");
+    t.addComponent(0, 2, c3);
+
+    assert.isTrue(Plottable.Component.Group.prototype.isPrototypeOf((<any> t)._rows[0][2]));
```

Comment:
Probably add a message to this `assert` statement, something like "A group was created". Consider that if this test fails all that the developer will see is "Expected false to be true".